### PR TITLE
[RELEASE] 1.2.2

### PR DESCRIPTION
**Pull request by joshua-stauffer**
release prep for 1.2.2

### CODE REVIEW

File: docs/docusaurus/docs/oss/changelog.md [hunk 1]
```diff
@@ -14,6 +14,29 @@ When we deprecate our public APIs, we will
 
 Before we completely remove the functionality in a new major release, there will be at least one minor release that contains the deprecation so that you can smoothly transition to the new API.
 
+### 1.2.2
+* [FEATURE] Add check for valid column type when calling add_batch_def in a sql asset ([#10590](https://github.com/great-expectations/great_expectations/pull/10590))
+* [FEATURE] Expectations tests against SQL backends infer column types ([#10622](https://github.com/great-expectations/great_expectations/pull/10622))
+* [BUGFIX] Parse spaces in row condition column name ([#10611](https://github.com/great-expectations/great_expectations/pull/10611)) (thanks @ashmortar)
```
Comment (cdkini) [inline docs/docusaurus/docs/oss/changelog.md]: This is Aaron - we should remove this attribution and update `teams.yml`

Comment (joshua-stauffer) [reply docs/docusaurus/docs/oss/changelog.md]: thanks, missed this 🙇

Comment (billdirks) [inline docs/docusaurus/docs/oss/changelog.md]: Can you remove the thanks here and add Aaron to .github/teams.yml is a separate PR?

Comment (joshua-stauffer) [reply docs/docusaurus/docs/oss/changelog.md]: thanks, missed this 🙇 but you and Chetan didn't!

**Review by cdkini — APPROVED** [review]

**Pull request merged**